tasks/gulp/compile-assets.js

```diff
@@ -31,9 +30,20 @@ const errorHandler = function (error) {
   this.once('finish', () => process.exit(1))
   this.emit('end')
 }
+// used only for review app as they include showing of media queries
```

Comment:
The existing app assets seem to be getting compiled OK, so not sure why this is needed.